### ISSUE TITLE
Reading: Stolen Probability: A Structural Weakness of Neural Language Models

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#111]
## 0. Paper
@inproceedings{demeter-etal-2020-stolen,
    title = "Stolen Probability: A Structural Weakness of Neural Language Models",
    author = "Demeter, David  and
      Kimmel, Gregory  and
      Downey, Doug",
    booktitle = "Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics",
    month = jul,
    year = "2020",
    address = "Online",
    publisher = "Association for Computational Linguistics",
    url = "https://www.aclweb.org/anthology/2020.acl-main.198",
    doi = "10.18653/v1/2020.acl-main.198",
    pages = "2191--2197",
    abstract = "Neural Network Language Models (NNLMs) generate probability distributions by applying a softmax function to a distance metric formed by taking the dot product of a prediction vector with all word vectors in a high-dimensional embedding space. The dot-product distance metric forms part of the inductive bias of NNLMs. Although NNLMs optimize well with this inductive bias, we show that this results in a sub-optimal ordering of the embedding space that structurally impoverishes some words at the expense of others when assigning probability. We present numerical, theoretical and empirical analyses which show that words on the interior of the convex hull in the embedding space have their probability bounded by the probabilities of the words on the hull.",
}
